pkg/cmd/server/start/start_master.go

```diff
@@ -580,6 +582,29 @@ func checkForOverrideConfig(ac configapi.AdmissionConfig) (*overrideapi.ClusterR
 	return overrideConfig, nil
 }
 
+type GenericResourceInformer interface {
+	ForResource(resource schema.GroupVersionResource) (kinformers.GenericInformer, error)
+}
+
+type genericInformers struct {
```

Comment:
This should be in a separate package.  I agree that combining informers is useful, but I suspect we'll need it more generally.